k8s-operator/apis/v1alpha1/types_proxygroup.go

```diff
@@ -111,6 +111,11 @@ type TailnetDevice struct {
 	// assigned to the device.
 	// +optional
 	TailnetIPs []string `json:"tailnetIPs,omitempty"`
+
+	// StaticEndpoints are the endpoints `<IP>:<port>` set on the device to facilitate direct
+	// connections
```

Comment:
```suggestion
StaticEndpoints are user configured, 'static' endpoints by which tailnet peers can reach this device.
```

Or something like this- since this is on status, it makes more sense to talk about what exactly this is, the 'can facilitate direct connections' is more for the spec part where users will be looking what else to configure